ethereum-common/src/beacon/light.rs

```diff
@@ -1,4 +1,4 @@
-// Module contains lightened versions of the entities.
+//! Module contains lightened versions of the entities.
```

Comment:
Please clarify here what means `lightened` I suppose it means that some merkle sub-trees are precomputed?

Reply:
Good catch, thanks! Will add the note